draft-ietf-quic-transport.md

```diff
@@ -192,14 +192,10 @@ QUIC packet:
 
 Endpoint:
 
-: An entity that can participate in a QUIC conversation by generating,
+: An entity that can participate in a QUIC connection by generating,
```

Comment:
```suggestion
: An entity that participates in a QUIC connection by generating,
```

Reply:
I thought about `s/can participate/participates/` but then I realized that during the handshake the endpoints send and receive QUIC packets before the connection is established -- perhaps that's why it was written this way.  I also want to limit this change only to the connection/conversation business.